docs/source/welcome_to_sycamore/get_started.md

```diff
@@ -95,9 +98,9 @@ Once the data has been loaded, some sample questions to ask are:
 
 ## Demo: Ingest and Query Data From An Arbitrary Website
 
-You can crawl and ingest an arbitrary website with the Sycamore default data ingestion code. However, this code is not not optimized for any given dataset, so the answer quality may vary on new websites.
+You can crawl and ingest a website with the Sycamore default data preparation code. However, this code is not not optimized for any given dataset, so the answer quality may vary on websites that have different characteristics.
 
-Note: Processing data using the default code will send your data to OpenAI, and optionally Amazon Textract, for calls to their AI services. Consider whether this is acceptable if you are using a non-public website for testing.
+*Note: Processing data using the default code will send your data to OpenAI, and optionally Amazon Textract, for calls to their AI services. Consider whether this is acceptable if you are using a non-public website for testing.*
```

Comment:
```
> [!NOTE]
....
```
or perhaps better [!WARNING]
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts

Reply:
Ack